pkg/pb/overlay.proto

```diff
@@ -85,6 +86,16 @@ message QueryResponse {
 message PingRequest {};
 message PingResponse {};
 
+// TODO: add fields that validate who is requesting the info
+message InfoRequest {}
+
+message InfoResponse {
+    bytes id = 1 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
+    node.NodeType type = 2;
+    node.NodeMetadata metadata = 3;
+    node.NodeRestrictions restrictions = 4;
```

Comment:
Is there a better name for this?

Reply:
Maybe there is a better name, but `NodeRestrictions` is an already defined type from the past. It's just referenced here.

Reply:
I'm aware that these are already defined, it's more of we can improve the naming more easily right now, instead of later when it's integrated into elsewhere.

Reply:
Also completely fine to handle it later.

Reply:
How about `NodeCapacity` instead of `NodeRestrictions`?

Reply:
Yes, that's a really nice name.

Reply:
Done.